src/venue.md

```diff
@@ -77,3 +79,78 @@ The [Malacuna Hostel] (previously known as [Hilton Chambers]) is also worth cons
 [Hilton Chambers]: https://maps.app.goo.gl/C5uXMdBcdpzUfUHm8
 
 [Northern Quarter]: https://www.visitmanchester.com/things-to-see-and-do/explore/neighbourhoods/northern-quarter/
+
+## Getting here
+
+### By Train
+
+Manchester has three major city-centre train stations:
+
+- **Manchester Piccadilly** – a **20-minute walk, 10 minutes by taxi**, or **10–15 minutes by bus or Beryl bike**
+- **Manchester Oxford Road** – a **10-minute walk** to Contact Theatre, or **3 minutes by taxi**
+- **Manchester Victoria** – approximately **30 minutes by public transport** or **15–20 minutes by taxi**
+
+To get to Contact Theatre from **Piccadilly**, you can:
+
+- Take a **bus from Piccadilly Gardens** towards Oxford Road and get off at the **University of Manchester**
+- Rent a **Beryl bike or e-bike**
+- Take a **taxi or rideshare** directly to the venue
+
+### By Car
+
+**Contact Theatre does not have public parking**. The adjacent car park is for **permit holders only**, but nearby options include:
```

Comment:
I know we mention elsewhere that it costs* £70 a day to drive in the area between certain hours, but it might be worth mentioning it also here, or briefly mentioning the charge and linking to it.

*- yes, technically it's a fine.

Reply:
(I read these in the wrong order, so this might already have been done)